.gitignore

```diff
@@ -1,5 +1,7 @@
 /bin
 /build
+/build-debug
+/build-release
```

Comment:
Can you revert his file?   The wabt Makefile uses subdirectories or `out/..` , I you use a different place for your output you can always add these to your `$HOME/.gitignore` so it effects all your projects.

Reply:
Reverted.

Reply:
can use `$GIT_DIR/info/exclude` too, see https://git-scm.com/docs/gitignore (`man gitignore`)